tests/runmode_test/test.yaml

```diff
@@ -0,0 +1,16 @@
+requires:
+  min-version: 7 
+  pcap: false
+
+exit-code: 1
+args:
+   - --dump-config
```

Comment:
This is not needed for the test, it was an example given by Juliana. The args should be just like mentioned in the redmine ticket.